src/NHibernate.Validator/Mappings/OpenClassMapping.cs

```diff
@@ -78,41 +78,41 @@ public void AddConstraint(FieldInfo field, Attribute attribute)
 		{
 			if (field == null)
 			{
-				throw new ArgumentNullException("field");
+				throw new ArgumentNullException(nameof(field));
 			}
 			if (attribute == null)
 			{
-				throw new ArgumentNullException("attribute");
+				throw new ArgumentNullException(nameof(attribute));
 			}
 			AddMemberConstraint(field, attribute);
 		}
 
 		public void AddMemberConstraint(MemberInfo member, Attribute attribute)
 		{
-			List<Attribute> constraints;
-
-			if (!membersAttributesDictionary.TryGetValue(member, out constraints))
+			if (!membersAttributesDictionary.TryGetValue(member, out var constraints))
 			{
 				constraints = new List<Attribute>();
 				membersAttributesDictionary.Add(member, constraints);
 			}
-			Attribute found = constraints.Find(x => x.TypeId.Equals(attribute.TypeId));
+			var found = constraints.Find(x => x.TypeId.Equals(attribute.TypeId));
 			if (found == null || AttributeUtils.AttributeAllowsMultiple(attribute))
 			{
+				constraints.Add(attribute); 
+				//Not possible to stop adding constraints here based on intersecting tags, because Tags not yet assigned when this code is executed

```

Comment:
Can you clarify what happens in that case? Is the fluent configuration checked and "cleaned up" later anyway? Or does this cause the overriding to not occur in this case, letting all attributes applied even if contradictory?